### DISCUSSION TITLE
Why do I have to pass my make payment POST request through the back end?

### DISCUSSION THREAD

**Discussion by Pacocc97** [mercadopago/sdk-react#58] [Q&A] [answered]
Hello my people. Im wondering why when I do a post request for a payment, I have to pass through my back end and cant make it directly from my front end to the mercado pago api?
Why I need to do this request directly from my front end if I´ll end sendig the exact same data.

**Comment by lemazzo** [comment, marked as answer]
Hello, @Pacocc97! Your question was answered at [mercadopago/sdk-js#168](https://github.com/mercadopago/sdk-js/discussions/168). Thanks!
